modules/helm-creating-a-custom-helm-chart-on-openshift.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="creating-a-custom-helm-chart-on-openshift_{context}"]
+= Creating a custom Helm chart on {product-title}
+
+.Procedure
+. Create a new project:
++
+----
+$ oc new-project nodejs-ex-k
+----
+
+. Download an example Node.js chart that contains {product-title} objects:
++
+----
+$ git clone https://github.com/redhat-developer/redhat-helm-charts/tree/tech-preview-4.3/alpha/nodejs-ex-k
```

Comment:
This clone command does not work:
```
$ git clone https://github.com/redhat-developer/redhat-helm-charts/tree/tech-preview-4.3/alpha/nodejs-ex-k
Cloning into 'nodejs-ex-k'...
fatal: repository 'https://github.com/redhat-developer/redhat-helm-charts/tree/tech-preview-4.3/alpha/nodejs-ex-k/' not found
```

Reply:
This should be a clone and 'cd' into the right directory.